recipes-kernel/linux/linux-toradex-kmeta.inc

```diff
@@ -1,10 +1,13 @@
-SRCREV_meta = "99e38525088a105d6e7d99bd0eb07c9880606d64"
-SRCREV_meta:use-head-next = "${AUTOREV}"
+SRCREV_torizon-meta = "99e38525088a105d6e7d99bd0eb07c9880606d64"
```

Comment:
Are you not bumping the commit hash yet to the latest on the scarthgap branch?

Reply:
I could bump, but since we're only building integration for now, we would only be using `AUTOREV`. The hash will only come into play when we do our first release, and we already have a task to bump the hashes there.